openstack/compute/v2/servers/fixtures.go

```diff
@@ -1,7 +1,7 @@
-// +build fixtures
```

Comment:
The build tag has to be at the top. Try using 
`go test -tags fixtures github.com/rackspace/gophercloud/openstack/compute/v2/servers`
to run those tests.